src/Apps/ViewingRoom/ViewingRoomApp.tsx

```diff
@@ -0,0 +1,438 @@
+import React, { useState } from "react"
+import { AppContainer } from "Apps/Components/AppContainer"
+import {
+  color,
+  Flex,
+  Box,
+  ResponsiveImage,
+  Sans,
+  Spacer,
+  ReadMore,
+  Serif,
+  Color,
+  FontWeights,
+  Image,
+  Button,
+  ChevronIcon,
+  ProgressBar,
+} from "@artsy/palette"
+
+const ViewingRoomApp: React.FC = props => {
+  const [selectedTab, setTab] = useState<"statement" | "works">("statement")
+
+  const showWorks = () => {
```

Comment:
[Again, just noting for later] but I think we'll want clicking "works" to take you to a new URL.